site/mkdocs.yml

```diff
@@ -75,21 +75,26 @@ nav:
     - Maintenance: maintenance.md
     - Performance: performance.md
     - Reliability: reliability.md
-  - Engines:
-    - Spark:
-      - Getting Started: getting-started.md
-      - Configuration: spark-configuration.md
-      - DDL: spark-ddl.md
-      - Queries: spark-queries.md
-      - Writes: spark-writes.md
-      - Maintenance Procedures: spark-procedures.md
-      - Structured Streaming: spark-structured-streaming.md
-    - Flink:
-      - Getting Started: flink.md
-      - Flink Connector: flink-connector.md
-    - Hive: hive.md
-    - Trino: https://trino.io/docs/current/connector/iceberg.html
-    - PrestoDB: https://prestodb.io/docs/current/connector/iceberg.html
+  - Spark:
+    - Getting Started: getting-started.md
+    - Configuration: spark-configuration.md
+    - DDL: spark-ddl.md
+    - Queries: spark-queries.md
+    - Writes: spark-writes.md
+    - Maintenance Procedures: spark-procedures.md
+    - Structured Streaming: spark-structured-streaming.md
+  - Flink:
+    - Getting Started: flink.md
+    - Flink Connector: flink-connector.md
+  - Hive:
```

Comment:
Hive isn't that long of a page right now. Why split it out in this commit rather than just leaving it as-is and having a high-level link to the page?

Reply:
This was really just to make the drop-downs look consistent but I'm realizing that issue partially goes away now that everything is flat. I'll consolidate the hive section back into a single file.

Reply:
Updated! Hive is now a single top level item that points to `hive.md` and renders the whole page when you click on it.

<img width="1780" alt="Screen Shot 2021-11-03 at 7 38 43 AM" src="https://user-images.githubusercontent.com/43911210/140081434-d9c19073-5d0d-434d-8654-a73c99ec68d9.png">
.